cypress/integration/pages/homePage/urls.js

```diff
@@ -0,0 +1,8 @@
+export default [
+  {
+    service: 'kyrgyz',
+    local: '/kyrgyz/tipohome',
+    test: '/kyrgyz/tipohome',
+    live: null,
```

Comment:
There will never be a live `/kyrgyz/tipohome` route, however, once we start to migrate these pages to `/kyrgyz` we can update the routes here, and remove / disable the frontPage routes in the settings.js file.

Reply:
How would we execute only one service when there are multiple service in this file?

Reply:
Ah, that's a good shout - I would need to incorporate the `CYPRESS_ONLY_SERVICE` functionality. I'll add that now.

Reply:
This is now done - feel free to try it out!